Library/Homebrew/cmd/fetch.rb

```diff
@@ -42,9 +42,15 @@ def fetch
   end
 
   def fetch_bottle?(f)
-    return true if ARGV.force_bottle? && f.bottle
-    return false unless f.bottle && f.pour_bottle?
-    return false if ARGV.build_from_source? || ARGV.build_bottle?
+    return false unless f.bottle
+    return true if ARGV.force_bottle?
```

Comment:
It checks in the line above whether there's a bottle and returns `false` if there is none. Once we are in this line, we know for sure there exists a bottle.